### ISSUE TITLE
Some issue

### ISSUE THREAD

**Issue by Ray12138ee** [zpliulab/GENELink#4]
In the process of reproducing the experiments of the paper, I found that the current main.py mainly evaluates the AUC/AUPR of the validation set during the training process.
I have some questions that:
1. Are the AUC/AUPR reported in the paper based on validation set or test set?
2. What is the final evaluation process if test set is used?

**Comment by zpliulab** [comment]
Thanks for pointing this out. `Code/main.py` only shows the training-time validation process, while `Demo/demo.py` contains the complete training/validation/test pipeline. The AUC/AUPR reported in the paper are based on the **test set** results in `Demo/demo.py`.
